src/lib/core/clock_lowres.c

```diff
@@ -0,0 +1,99 @@
+/*
+ * SPDX-License-Identifier: BSD-2-Clause
+ *
+ * Copyright 2010-2022, Tarantool AUTHORS, please see AUTHORS file.
+ */
+#include <signal.h>
+#include <string.h>
+#include <say.h>
+#include <pthread.h>
+#include "clock.h"
+#include "clock_lowres.h"
+
+#if defined(__APPLE__) || !defined(NDEBUG)
+/**
+ * A thread that initialized this module. Only owner thread
+ * is allowed to handle SIGALRM and use methods from this module.
+ */
+static pthread_t owner;
+
+/** Check if current thread is owner. */
+bool
+clock_lowres_thread_is_owner(void)
+{
+	return pthread_equal(owner, pthread_self());
+}
+#endif
+
+/** Old sigaction for SIGALRM. */
+static struct sigaction osa;
+
+/** Granularity of clock update. */
+static const struct timeval LOW_RES_GRANULARITY = {
+	.tv_sec = 0,
+	.tv_usec = 10 * 1e3,
+};
+
+double
+clock_lowres_getres(void)
+{
+	struct timeval ts = LOW_RES_GRANULARITY;
```

Comment:
Inconsistent naming:

- `LOW_RES_GRANULARITY` vs `clock_lowres_getres` (different prefixes, granularity vs resolution)
- `clock_lowres_getres` vs `clock_lowres_monotnic` (one has `get` another doesn't)
- `clock_monotonic_lowres` vs `low_res_monotonic_clock` (different prefixes, underscore between `low` and `res`)

Please try to stick to the same naming policy in the same file. I'd use the following names:

- `clock_monotonic_lowres()` -> `clock_lowres_monotonic()` (same prefix for all public functions)
- `clock_lowres_getres()` -> `clock_lowres_resolution()`
- `LOW_RES_GRANULARITY` -> `resolution` (strictly speaking, what we have here is resolution, not granularity; it's a static constant so we don't really need to use a prefix)
- `low_res_monotonic_clock` -> `clock_lowres_monotonic_clock_value`

Reply:
done